src/callbacks.rs

```diff
@@ -42,7 +43,12 @@ pub extern fn output_focus(output: WlcOutput, focused: bool) {
 pub extern fn output_resolution(output: WlcOutput,
                             old_size_ptr: &Size, new_size_ptr: &Size) {
     trace!("output_resolution: {:?} from  {:?} to {:?}",
-             output, *old_size_ptr, *new_size_ptr);
+           output, *old_size_ptr, *new_size_ptr);
+    // Update the resolution of the output and it's children
```

Comment:
thanks, for things like that that don't really matter, feel free to just change that yourself